mypy/types.py

```diff
@@ -447,6 +447,31 @@ def deserialize(cls, data: JsonDict) -> 'TypeVarType':
         )
 
 
+class SelfType(ProperType):
+    def __init__(self, instance: 'Instance', fullname: str, line: int = -1, column: int = -1) -> None:
+        super().__init__(line, column)
+        self.name = "Self"
```

Comment:
Is this going to be changed? If not, it looks like a good candidate for a class-level value.